.github/workflows/python-tests.yml

```
@@ -0,0 +1,51 @@
+    
+        strategy:
+            matrix:
+                python-version: ['3.9', '3.10', '3.11', '3.12', '3.13']
+        
+        steps:
+            - uses: actions/checkout@v4
+
+            - name: Set up Python ${{ matrix.python-version }}
+              uses: actions/setup-python@v5
+              with:
+                python-version: ${{ matrix.python-version }}
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 18 but found 16

(indentation)